docker-compose.ubuntu.yml

```diff
@@ -7,13 +7,25 @@ services:
 #    image: quay.io/centerforopenscience/fakecas:latest
 #    command: fakecas -host=0.0.0.0:8080 -dbaddress=172.19.0.7:27017
 
-#  wb:
-#    volumes:
-#      - ../waterbutler:/code
+# wb:
+#   volumes:
+#     - ../../wb/waterbutler:/code
```

Comment:
Minor: can you set these and the other volume relative paths back to `../waterbutler` and `../modular-file-renderer`?